src/test/java/uk/gov/register/functional/ApplicationTest.java

```diff
@@ -30,7 +30,7 @@ public void setup() {
     @Test
     public void appSupportsCORS() {
         String origin = "http://originfortest.com";
-        Response response = register.target().path("/entries")
+        Response response = register.targetRegister("address").path("/entries")
```

Comment:
This is the same as `getRequest()` do think we should keep only one of them? maybe possibly or make a convention when to use what?